### ISSUE TITLE
Fix Renovate bot configuration

### ISSUE THREAD

**Issue by jooola** [libretime/libretime#2017]
Docusaurus v2 has been released and renovate hasn't proposed a PR yet.

I'll open a PR with packages upgrades.

**Comment by jooola** [comment]
Reopening as renovate didn't propose the docusaurus v2.1.0 yet, and we might need to report this upstream.

**Comment by jooola** [comment]
Does not look like  #2101 fixed this.

**Comment by jooola** [comment]
So after reading the docs and some testing, it turns out this is actually the expected behavior for renovate bot.

It does check if the dependencies in `website/packages.json` are up to date (within version range) but will not update the lock files if an newer version (again within version range) is available. So we will keep install the older version based on the lock file until we update it.

There is a [lockfilemaintenance](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance) option that can be enable to update the lock files.

So maybe we should enable this option to refresh the lock files once a week ?

**Comment by jooola** [comment]
Another option that we should consider is the dependencies dashboard:

We have hit the limit of concurrent renovate branches, which prevents renovate from propose newer updates for other packages. Since most of the dependencies are currently on hold, maybe we should move them to a dependency dashboard and leave space for other updates. We could also simply close them, but we have to track them somewhere.

If I understand renovate properly, reopening a old PR (once we know it is unblocked) is as simple as selecting it in the dependency dashboard or reopening the closed PR.

We can also raise the branchConcurrentLimit option to something >10.
